geomesa-jobs/README.md

```diff
@@ -65,6 +65,43 @@ geomesa> mvn clean install -P assemble -pl geomesa-jobs
 The following instructions assume you have built a shaded jar; if not you will need to use the 'libjars'
 argument to ensure the correct jars are available on the distributed classpath.
 
+#### Analytic Histograms
+
+GeoMesa provides the ability to analyze your data through histograms.
+
+The job can be invoked through yarn as follows (jar version may vary slightly):
+
+```shell
+geomesa> yarn jar geomesa-jobs/target/geomesa-jobs-accumulo1.5-1.0.0-shaded.jar \
```

Comment:
There's quite a lot of functionality that we could expose if we supported Hive - it has ```SELECT AS``` and ```GROUPBY``` and ```COUNT DISTINCT```.  Thoughts on looking into supporting Hive?